django-backend/fecfiler/web_services/summary/test_summary.py

```diff
@@ -15,11 +15,13 @@ def test_calculate_summary(self):
         f3x = F3XSummary.objects.get(id="b6d60d2d-d926-4e89-ad4b-c47d152a66ae")
         summary_service = SummaryService(f3x)
         summary = summary_service.calculate_summary()
-        self.assertEqual(summary["line_15"], Decimal("2125.79"))
-        self.assertEqual(summary["line_12"], Decimal("1212.12"))
+        self.assertEqual(summary["a"]["line_12"], Decimal("1212.12"))
+        self.assertEqual(summary["a"]["line_17"], Decimal("1000.00"))
+        self.assertEqual(summary["a"]["line_15"], Decimal("2125.79"))
+        self.assertEqual(summary["a"]["line_15"], summary["a"]["line_37"])
```

Comment:
we'll want unit tests for column b especially with this being our first

Reply:
Took me forever to figure out why the unit tests would not pick up any Column B calculations while the front-end worked fine.  The solution was to add dates to all the transactions in the test fixture :facepalm: